### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by DebyGrey**
## Awesome books: plain JavaScript with objects

In this project, I:

- collaborated with @God-Logic
- set up a development environment with linter checkers
- created a collection that keeps a list of books
- created a function to add a new book to the collection, with title and author
- created a function to remove a book from the collection
- created a function to display all books saved in the collection at the top part of the page.
- made sure that data is preserved in the browser's memory by using local storage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Deborah Fashoro**
+
+- GitHub: [@githubhandle](https://github.com/DebyGrey)
+- Twitter: [@twitterhandle](https://twitter.com/Deby_grey)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/deborah-fashoro)
+
```
Comment (omar-muhamad) [inline README.md:118]: - [x] Kindly add your collaborator @God-Logic name and social links here in the authors section to make your readme file more professional.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @DebyGrey and @God-Logic,

### Nice job, you are almost there.

giphy (2)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- [x] Your code is elegant but it's not a good practice to overuse comments in your code, It would be better if you only add comments for the complex parts of code that need clarify or for example the general use of certain functions.

image


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Deborah Fashoro**
```
Comment (tugsem) [inline README.md:113]: - [ ] As the previous reviewer required, kindly add your collaborator's name and social media in your readme file.

**Review by tugsem — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️
Hello @DebyGrey  and @God-Logic 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @DebyGrey and @God-Logic,

✔️ Descriptive documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**